### ISSUE TITLE
Emoji with U+fe0f match only first character

### ISSUE THREAD

**Issue by jcubic** [mathiasbynens/emoji-regex#43]
Another Emoji that don't match properly:

* ☝️ Index Pointing Up U+261d U+fe0f

it match only first character ☝

**Comment by mathiasbynens** [comment]
@tonton-pixel You’re absolutely right! The clearest way to express this is by using a not-yet(?)-standard RegExp feature, as described here: https://github.com/tc39/proposal-regexp-unicode-sequence-properties#matching-all-emoji-including-emoji-sequences

In other words, `emojiSequence` expands to what’s described here: https://github.com/tc39/proposal-regexp-unicode-sequence-properties#matching-emoji-sequences

```js
const reEmojiSequence = /\p{Emoji_Flag_Sequence}|\p{Emoji_Tag_Sequence}|\p{Emoji_ZWJ_Sequence}|\p{Emoji_Keycap_Sequence}|\p{Emoji_Modifier_Sequence}/u;
```

**Comment by mathiasbynens** [comment]
Actually, the `Emoji_Modifier_Base` comment is wrong. `<% emojiSequence %>` includes `\p{Emoji_Modifier_Sequence}`, but an `\p{Emoji_Modifier_Base}` symbol that is NOT followed by a `\p{Emoji_Modifier}` symbol doesn’t form a sequence, but it’s still an emoji. Here are some examples:

'☝⛹✊✋✌✍🎅🏃🏄🏊🏋👂👃👆👇👈👉👊👋👌👍👎👏👐👦👧👨👩👮👰👱👲👳👴👵👶👷👸👼💁💂💃💅💆💇💪🕵🕺🖐🖕🖖🙅🙆🙇🙋🙌🙍🙎🙏🚣🚴🚵🚶🛀🤘🤙🤚🤛🤜🤝🤞🤦🤰🤳🤴🤵🤶🤷🤸🤹🤼🤽🤾'

So we need `\p{Emoji_Modifier_Sequence}` (as included in `emojiSequence`) but in addition, we need `\p{Emoji_Modifier_Base}`.

**Comment by mathiasbynens** [comment]
Note that U+261D has both `\p{Emoji_Modifier_Base}` and `\p{Emoji}`.
